### ISSUE TITLE
Funds in wallet and payments from wallet stuck as 'confirming'

### ISSUE THREAD

**Issue by boswell12** [bitpay/copay#9720]
My wallet in Copay legacy was debited with an amount over a week ago, and have been using it fine since then. Now however, I am told that 90%+ of the fund in my wallet are 'confirming' so cannot be used. However, the balance was confirmed over a week ago so I do not understand why this is happening.

In addition, I have tried sending some payments out of this wallet and they too are frozen as 'confirming' despite me setting the fees to normal

**Comment by boswell12** [comment]
I am still unable to use the funds in my wallet, any advice as to how to resolve this? I suspect it is due to the payments out of the wallet not being confirmed (although they were for a fraction of the total funds in the wallet), so I might need to know how to get them confirmed or deleted?

**Comment by boswell12** [comment]
These payments are still stuck and i remain effectively locked out of my wallet, any advice on how to delete these payments would be much appreciated

**Comment by Gennttii** [comment]
maybe you have put low fee in your transaction, share your BTC address and let me check

**Comment by boswell12** [comment]
Hi, the transactions finally cleared and my wallet became unfrozen yesterday, so I guess the fees were too low given the amount of activity on the blockchain in the last few days.

The fact is though I set the fees to 'normal' in copay, which always leads to confirmations within about half an hour or so, and the dollar amount I was charged was actually quite high. Therefore it's pretty frustrating that they got stuck but I guess I just got unlucky.
